src/exe/colmap.cc

```diff
@@ -300,10 +303,30 @@ int RunStereoFuser(int argc, char** argv) {
   fuser.Start();
   fuser.Wait();
 
-  std::cout << "Writing output: " << output_path << std::endl;
-  WriteBinaryPlyPoints(output_path, fuser.GetFusedPoints());
-  mvs::WritePointsVisibility(output_path + ".vis",
-                             fuser.GetFusedPointsVisibility());
+  Reconstruction reconstruction;
+
+  // read data from sparse reconstruction
+  if (workspace_format == "colmap" )
+    reconstruction.Read(JoinPaths(workspace_path, "sparse"));
+
+  // overwrite sparse point cloud with dense point cloud from fuser
+  reconstruction.ImportPLY(fuser.GetFusedPoints());
+
+  // write output
+  StringToLower(&output_type);
+  if (output_type == "bin") {
+    reconstruction.WriteBinary(output_path);
+  } else if (output_type == "txt") {
+    reconstruction.WriteText(output_path);
+  } else if (output_type == "ply") {
+    std::cout << "Writing output: " << output_path << std::endl;
```

Comment:
Extract this to before the if

Reply:
I didn't which line you meant. I moved the 'std::cout ...' before the if.